### DISCUSSION TITLE
LambdaExpression.CompileToMethod missing in .net core.

### DISCUSSION THREAD

**Discussion by binarysoftdev** [dotnet/runtime#88555] [Q&A]
I am trying to move some of my code from .Net 4.8 to .Net core 6.   I have a lot of code the uses LambdaExpression.CompileToMethod.   Does anyone have a replacement for this method?

**Comment by huoyaoyuan** [comment]
See also #19943 #20270 https://github.com/dotnet/runtime/issues/62956#issuecomment-997168810
